compose/build.gradle.kts

```diff
@@ -73,6 +74,23 @@ dependencies {
     debugImplementation(libs.androidx.ui.test.manifest)
 }
 
+val useVulkan = project.hasProperty("useVulkan")
```

Comment:
I took a shortcut by doing it here since this is the only module that depends on maplibre11. If we have other modules in the future we should move this to a Gradle plugin.

Reply:
I think this works! This is the only module where we depend on MapLibre in this project.